script.js

```diff
@@ -87,3 +87,36 @@ document.addEventListener('DOMContentLoaded', () => {
   const load = new Book();
   load.loadFromStorage();
 });
+
+// nav bar
+
+const bookList = document.querySelector('.book-list');
+const addList = document.querySelector('.add-book');
+const contactList = document.querySelector('.contact-list');
+
+const allSection = document.getElementById('book-list-container');
+const inputSection = document.getElementById('form');
+const contactSection = document.querySelector('.contact');
+
+bookList.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  allSection.style.display = 'block';
+  inputSection.style.display = 'none';
+  contactSection.style.display = 'none';
+});
+
+addList.addEventListener('click', (e) => {
+  e.preventDefault();
+  allSection.style.display = 'none';
+  inputSection.style.display = 'block';
+  contactSection.style.display = 'none';
+  document.querySelector('.adding-books').style.display = 'block';
+});
+
+contactList.addEventListener('click', (e) => {
+  e.preventDefault();
+  allSection.style.display = 'none';
+  inputSection.style.display = 'none';
+  contactSection.style.display = 'flex';
+});
```

Comment:
- [ ] Using [the wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md#project-requirements) as a guide the items on each page should be centered as in the wireframe layout. Kindly improve your app layout to match the design.

  form page | contact page
  :---:|:---:
  ![add-new](https://user-images.githubusercontent.com/54848363/226922700-29960235-ff59-45da-9f2d-b5f594f98fc2.jpg) | ![cont](https://user-images.githubusercontent.com/54848363/226922704-9d1f5d21-5a9e-436c-9cbe-c7982f18995f.jpg)